docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Build a Mobile App for Token Staking on Fuse.
+sidebar_position: 16
+---
+
+# Build a Mobile App for Token Staking on Fuse.
+In this guide, we will walk through the steps to build a mobile app using the Fuse Flutter SDK method [Token Staking](https://pub.dev/packages/fuse_wallet_sdk#staking). This guide will walk you through setting up your Flutter environment, initializing the Fuse SDK, and implementing wallet creation and token staking functionalities. Let’s dive right in!
+
+## Pre-requites:
+
+- Basic knowledge of Dart/Flutter
+- An API Key from the [Console](https://console.fuse.io/build)
+
+## Set Up Your Flutter Environment
+First things first, you need to have Flutter installed on your machine. If you haven’t installed it yet, you can find the installation guide on the Flutter official website. Once you have Flutter set up, create a new project by running:
+
+```bash
+flutter create fuse_wallet_app
+cd fuse_wallet_app
+```
+
+Next, add the Fuse Wallet SDK to your project. Open your pubspec.yaml and add the following line under dependencies:
+
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  fuse_wallet_sdk: ^0.3.3
+```
+
+Check for the latest version on [pub.dev](https://pub.dev/packages/fuse_wallet_sdk)
+
+To install the new dependency, run 
+
+```bash
+flutter pub get
+```
+
+## Step 1: Initializing the Fuse SDK
+Now, let's initialize the Fuse SDK. Open your `main.dart` file, and set up the FuseSDK object with your API and private keys. Remember to keep your private key secure and not expose it in your codebase.
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:fuse_wallet_sdk/fuse_wallet_sdk.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
I copied updated code and faced this errors: 
![Screenshot 2024-06-02 at 17 27 54](https://github.com/fuseio/fuse-docs/assets/68244817/a13ceb8c-d85f-4876-a50c-25ce523a255d)
![Screenshot 2024-06-02 at 17 27 57](https://github.com/fuseio/fuse-docs/assets/68244817/5ba10791-7be8-47a8-9678-126a02a0c7e9)

Reply:
Fixed.